internal/pkg/vm/actor/builtin/initactor/init.go

```diff
@@ -126,7 +126,7 @@ type invocationContext interface {
 	runtime.InvocationContext
 	AddressForNewActor() (address.Address, error)
 	CreateNewActor(addr address.Address, code cid.Cid) error
-	Message() *types.UnsignedMessage
+	OriginalMessage() *types.UnsignedMessage
```

Comment:
As discussed, let's say `LegacyMessage`. This is the immediate message, not the top-level one.